File: __tests__/__fixtures__/output/javascript/jquery/application-form-encoded.js
const settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://httpbin.org/anything",
  "method": "POST",
  "headers": {
    "content-type": "application/x-www-form-urlencoded"
  },
  "data": {
    "foo": "bar",
    "hello": "world"
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/swift/urlsession/fixtures/application-form-encoded.swift
import Foundation

let parameters = [
  "foo": "bar",
  "hello": "world",
]
let joinedParameters = parameters.map { "\($0.key)=\($0.value)" }.joined(separator: "&")
let postData = Data(joinedParameters.utf8)

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.timeoutInterval = 10
request.allHTTPHeaderFields = ["content-type": "application/x-www-form-urlencoded"]
request.httpBody = postData

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))